test.js

```diff
@@ -64,19 +70,24 @@ describe('Run a voting session', () => {
     .send({ 
       payload: JSON.stringify({
         channel: { id: 1 },
-        actions: [{ value: 'simple' }],
+        actions: [{ value: 'Medium' }],
         user: {},
         original_message: { text: 'Feature 1' }
       })
     })
     .end((err, res) => {
       res.should.have.status(200)
       res.should.be.json
+      res.body.attachments[0].actions[0].value.should.equals('Simple')
+      res.body.attachments[0].actions[1].value.should.equals('Medium')
+      res.body.attachments[0].actions[2].value.should.equals('Hard')
+      res.body.attachments[0].actions[3].value.should.equals('reveal')
+
       done()
     })
   })
 
-  it('Reveal the results', () => {
+  it('Reveal the results', (done) => {
```

Comment:
It's very important writting a failing test first and to ensure that it really fails!

Ommiting that `done` argument will make the test to pass anyways.